runtime/v1/shim/service.go

```diff
@@ -553,8 +553,9 @@ func shouldKillAllOnExit(bundlePath string) (bool, error) {
 	if err != nil {
 		return false, err
```

Comment:
@fuweid Do you think this err should be removed as well?

Reply:
Yeah. I think we can just log it and return true.

Reply:
@fuweid @AkihiroSuda 
Please take another look.

Reply:
the commit seems still unchanged

Reply:
Changed this time, @AkihiroSuda